### PR TITLE
Add LS Dyna unit test

### LINKED ISSUE #581: Add unit test for LSDyna
I suggest documenting this, and adding a unit test once fixed

_Originally posted by @koubaa in https://github.com/ansys/pymechanical-examples/issues/52#issuecomment-1618946214_

### PR DESCRIPTION
**Pull request by dipinknair**
Simple unit test to check LS Dyna system in Mechanical
 Closes #581

### CODE REVIEW

**Review by sbradrick — APPROVED** [review]

File: .codespellrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [codespell]
 # In the event of a false positive, add the problematic word, in all lowercase, to a comma-separated list here:
-ignore-words-list = ans
\ No newline at end of file
+ignore-words-list = ans, globaly
```
Comment (koubaa) [inline .codespellrc:3]: this is a typo

File: tests/embedding/test_qk_eng_wb2.py [hunk 2]
```diff
@@ -291,10 +292,81 @@ def test_image_export(printer, selection, embedded_app):
...
-    # settings_720p.Capture = Ansys.Mechanical.DataModel.Enums.GraphicsCaptureType.ImageOnly
     settings_720p.Height = 720
     settings_720p.CurrentGraphicsDisplay = False
     ExtAPI.Graphics.ExportImage(
         os.path.join(os.getcwd(), "geometry.png"), image_export_format, settings_720p
     )
     assert os.path.isfile(os.path.join(os.getcwd(), "geometry.png"))
+
+
+@pytest.mark.embedding
```
Comment (koubaa) [inline tests/embedding/test_qk_eng_wb2.py:303]: put this in another file, not `test_qk_eng_wb2.py`.  This file is intended to contain the port of some QA tests to pymechanical.

File: tests/embedding/test_qk_eng_wb2.py [hunk 3]
```diff
@@ -291,10 +292,81 @@ def test_image_export(printer, selection, embedded_app):
...
+@pytest.mark.minimum_version(241)
```
Comment (koubaa) [inline tests/embedding/test_qk_eng_wb2.py:304]: this needs to be marked as windows only, we have a limitation for LSDYNA on linux embedding right now

**Review by koubaa — CHANGES_REQUESTED** [review]
see my comments

File: .codespellrc (hunk 1 above)
Comment (dipinknair) [reply .codespellrc:3]: This is to ignore `GlobalY` in test

File: tests/embedding/test_analysis.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
```
Comment (koubaa) [inline tests/embedding/test_analysis.py:1]: I think a better name for this file is test_dyna.py

File: tests/embedding/test_analysis.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+except:
+    # No embedding - this import breaks test collection
+    global_variables = {}
+
+from .test_qk_eng_wb2 import get_assets_folder
+
+
+@pytest.mark.embedding
+@pytest.mark.windows_only
+def test_lsdyna(printer, selection, embedded_app):
```
Comment (koubaa) [inline tests/embedding/test_analysis.py:40]: let's be more specific about the type of the test. How would you describe it?

**Review by koubaa — APPROVED** [review]

File: tests/embedding/test_qk_eng_wb2.py [hunk 6]
```diff
@@ -274,7 +275,7 @@ def test_image_export(printer, selection, embedded_app):
 
     Testing offscreen image export
     """
-    globals().update(global_variables(embedded_app))
+    globals().update(global_variables(embedded_app, True))
```
Comment (koubaa) [inline tests/embedding/test_qk_eng_wb2.py:278]: if True is used here, I would expect to see shortened namespaces elsewhere

**Review by koubaa — APPROVED** [review]

**Review by sbradrick — APPROVED** [review]

File: tests/embedding/test_dyna.py [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+@pytest.mark.windows_only
+def test_lsdyna(printer, embedded_app):
+    """
+    Unit test for LSDyna.
+
+    This test focuses on LSDyna analysis in Mechanical.
+    Runs only in Windows environment. This test involves
+    a simple geometry with high velocity
+    hitting on rigid wall.
+    """
```
Comment (sbradrick) [inline tests/embedding/test_dyna.py:48]: This test focuses on a LSDyna analysis in Mechanical for the Windows environment only. This test involves a simple geometry with high velocity striking a rigid wall.

**Pull request merged**